consensus/core/src/config/bps.rs

```diff
@@ -33,7 +33,7 @@ impl<const BPS: u64> Bps<BPS> {
     }
 
     /// Returns the GHOSTDAG K value which was pre-computed for this BPS
-    /// (see [`calculate_ghostdag_k`] and [`gen_ghostdag_table`] for the full calculation)
+    /// (see [`calculate_ghostdag_k`] and `gen_ghostdag_table` for the full calculation)
```

Comment:
What's the logic for which text is surrounded by square brackets (`[]`)?

Reply:
Typescript Type/Interface forwarding

Reply:
This is a RustDoc syntax that creates intra-doc links to identifiers.  Identifiers must be visible within the Rust scope of the documentation or referred to via absolute paths (just like you would do when performing regular imports).

In this specific case, `gen_ghostdag_table` can not be referenced to as it is not in the scope accessible by this documentation block (as this is a reference to a unit test function that does not exist during normal compilation).